ReactQt/runtime/src/rootview.cpp

```diff
@@ -288,6 +309,25 @@ void RootView::sendMouseEvent(QMouseEvent* event, const QString& eventType, QQui
     event->setAccepted(true);
 }
 
+#ifdef RCT_DEV
+void RootView::loadDevMenu() {
+    QQmlComponent component(qmlEngine(this));
```

Comment:
This chunk of code is similar to `ViewManager::createView()`. Probably we can move common function to `utilities.h`